content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

```diff
@@ -0,0 +1,71 @@
+---
+permalink: "rackspace-fabric-faq"
+audit_date: '2022-03-17'
+type: article
+title: "Rackspace Fabric FAQ"
+created_date: '2022-03-17'
+created_by: Asmita Nakwa
+last_modified_date: '2022-03-17'
+last_modified_by: Asmita
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+### General
+{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}
+
+If this is cloud account admin, they can review the following link (https://account.rackspace.com/event-history). It should help them understand any recent changes/access to their account.
+{{</accordion>}}
+
+{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}
+
+The customer needs to add the firewall as a device and the **Add VPN User** template will be visible in common requests.
+{{</accordion>}}
+
+{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}
+
+MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team.
+{{</accordion>}}
+
+{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}
+
+The customer will get notification only if the device in question is opted in.{{</accordion>}}
+
+{{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}
+
+There is no primary contact on the account. That is required for the account to work. I believe it can be set from Core.
```

Comment:
Check with Punit on this one if you haven't done so already.
This `I believe it can be set from CORE` line might not be useful to the customer. They don't use CORE. Dedicated users would adjust permissions in MyRack. Cloud users would adjust in Apollo or Account sections. So we want to explain where in the customer portal that a customer can change. We could even say for them to have an Admin from the customer side can update it, or instruct them to follow up with their support or their Account Manager.

Reply:
Updated as Create a new ticket using fabric ticketing application under the category **Account** and select account type as **User Management**.